hazelcast/src/main/java/com/hazelcast/config/AbstractCacheConfig.java

```diff
@@ -89,8 +89,11 @@
      */
     protected boolean isManagementEnabled;
 
+    // todo: to be removed
```

Comment:
Why not to remove in this PR?

Reply:
We will not be removing it in 5.0, but I thought leaving `// todo` trace might make it easier to find once we come to a point to completely remove Persistence and DataPersistence predecessors.